test-util/src/main/scala/io/buoyant/test/ActivityValues.scala

```diff
@@ -0,0 +1,37 @@
+package io.buoyant.test
+
+import com.twitter.util.Activity
+import org.scalatest.exceptions.TestFailedException
+
+/**
+ * Like TryValues but for Activity.
+ */
+trait ActivityValues {
```

Comment:
Would love a mini tutorial to see how this works with your unit tests!

Reply:
Added some docs in b891c92813823fb988a3ce3100421df66d1449cf that should help explain how this is useful – let me know if you have any further questions!